riva/clients/asr/streaming_recognize_client.cc

```diff
@@ -165,6 +165,11 @@ StreamingRecognizeClient::GenerateRequests(std::shared_ptr<ClientCall> call)
       config->set_enable_automatic_punctuation(automatic_punctuation_);
       config->set_enable_separate_recognition_per_channel(separate_recognition_per_channel_);
       auto custom_config = config->mutable_custom_configuration();
+      std::unordered_map<std::string, std::string> custom_configuration_map =
+          ReadCustomConfiguration(custom_configuration_);
+      for (auto& it : custom_configuration_map) {
+        (*custom_config)[it.first] = it.second;
+      }
       (*custom_config)["test_key"] = "test_value";
```

Comment:
the "test_key" and "test_value" should be unused and can be removed now, right? I think it was just put as a placeholder.

Reply:
yeah, will remove it 👍